notebooks/time-to-merge-prediction/workshop/deployment-config.yaml

```
@@ -0,0 +1,66 @@
+apiVersion: machinelearning.seldon.io/v1
+kind: SeldonDeployment
+metadata:
+  name: your-username
+spec:
+  name: your-username
```

can we make these capitalized so its clearer where to edit 🙂 
```suggestion
  name: YOUR-USERNAME
```